yang/frr-zebra.yang

```diff
@@ -1992,12 +2052,67 @@ module frr-zebra {
         type uint32 {
           range "1..100000";
         }
+        units "megabits/sec";
```

Comment:
We should just remove this bandwidth node. 

This is really a `config false`, which would then go under the zebra `state` container; however, there's already a `speed` in the `frr-interface` model which is also actually implemented. :)

Would be nice prior to 10.0 to fix the interface model `speed` node to be `bits/sec` and `uint64` no reason to restrict to other units.

Reply:
There's an existing `bandwidth` command that allows to set the bandwidth manually. I'm not sure that the code currently supports using the actual interface speed, so we can't just remove it. And also this will break backwards compatibility.

The operational speed is a separate thing. I agree it should be `bits/sec` and `uint64` (or maybe `gauge64` as in IETF interfaces model).

Reply:
I didn't see that it was used. However, there's no issue with backward compat and the zebra model.